### ISSUE TITLE
Chips component has a partial keyboard trap

### ISSUE THREAD

**Issue by rpd10** [angular/material2#4593]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
When using the keyboard to navigate the page, you should be able to tab into a chips component and use the arrow keys to navigate items, then tab or shift-tab to move away from the chips component.

#### What is the current behavior?
You are able to forward tab out of the component, but shift-tab does not let you leave and move back to the previous item on the page.

#### What are the steps to reproduce?
This is evident on the component guides page: https://material.angular.io/components/component/chips

1. Click on overview.
2. Tab into the chip
3. Use the arrow keys to navigate through chips, use space bar to select.  All good.
4. Forward tab moves you to the next focusable element on the page, good.
5. Shift-tab back to the chips.
6. Shift-tab again, you are unable to lose focus on the first chip and navigate back to the previous item on the page.
7. Use the arrow keys to select a chip other than the first. Shift tab, you are taken back to the first chip.  Shift-tab again, you are unable to lose focus.

Providing a Plunker (or similar) is the *best* way to get the team to see your issue.
Plunker template: https://goo.gl/DlHd6U

#### What is the use-case or motivation for changing an existing behavior?
Accessibility issue
